src/renderer/components/dialogs/MailtoDialog.tsx

```diff
@@ -99,37 +97,3 @@ export default function MailtoDialog(props: {
     </DeltaDialogBase>
   )
 }
-
-export async function doMailtoAction(chatId: number, messageText: string) {
```

Comment:
I renamed this method to `createDraftMessage` as I thought it is a more explicit name for what the method actually does. As it is only doing things in a chat context I moved it into the "chat methods" file. Happy to get some feedback on the naming and placing!